.travis.yml

```diff
@@ -1,13 +1,24 @@
 language: scala
 jdk: oraclejdk8
 scala:
-    - 2.11.8
-    - 2.12.1
+    - 2.11.12
+    - 2.12.4
 
 script:
-- sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport
+- sbt ++$TRAVIS_SCALA_VERSION -Dakka.test.timefactor=5 clean coverage test coverageReport
```

Comment:
I added this line because of this fail (https://travis-ci.org/cakesolutions/scala-kafka-client/jobs/319081822#L5338) at the unit test. I saw the suggestion from [here](https://github.com/akka/akka/issues/15515#issuecomment-48609368).

Reply:
👍